src/plugins/workspace/public/components/delete_workspace_modal/delete_workspace_modal.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFieldText,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+
+interface DeleteWorkspaceModalProps {
+  selectedItems: string[];
+  onClose: () => void;
+  onConfirm: () => void;
+}
+
+export function DeleteWorkspaceModal(props: DeleteWorkspaceModalProps) {
+  const [value, setValue] = useState('');
+  const { onClose, onConfirm, selectedItems } = props;
+
```

Comment:
There was a useEffect here to reset the input value, it should not have been deleted.

Reply:
In workspace overview page, I add a variable named "deleteWorkspaceModalVisible" to control the visibility of the delete modal:  
`{deleteWorkspaceModalVisible && <DeleteWorkspaceModal />}`
Therefore, the input value will be reset when the delete modal changes visibility.